cpp/include/nvtx3/nvtx3.hpp

```diff
@@ -240,43 +240,43 @@
  * lifetime of objects. Similar to `std::lock_guard` in the C++ Standard
  * Template Library.
  *
- * \subsection THREAD_RANGE Thread Range
+ * \subsection scoped_range Thread Range
  *
- * `nvtx3::domain_thread_range` is a class that begins a range upon construction
+ * `nvtx3::scoped_range_in` is a class that begins a range upon construction
  * and ends the range at destruction. This is one of the most commonly used
  * constructs in NVTX++ and is useful for annotating spans of time on a
  * particular thread. These ranges can be nested to arbitrary depths.
  *
- * `nvtx3::thread_range` is an alias for a `nvtx3::domain_thread_range` in the
+ * `nvtx3::scoped_range` is an alias for a `nvtx3::scoped_range_in` in the
  * global NVTX domain. For more information about Domains, see \ref DOMAINS.
  *
  * Various attributes of a range can be configured constructing a
- * `nvtx3::domain_thread_range` with a `nvtx3::event_attributes` object. For
+ * `nvtx3::scoped_range_in` with a `nvtx3::event_attributes` object. For
  * more information, see \ref ATTRIBUTES.
  *
  * Example:
  *
  * \code{.cpp}
  * void some_function(){
  *    // Creates a range for the duration of `some_function`
- *    nvtx3::thread_range r{};
+ *    nvtx3::scoped_range r{};
  *
  *    while(true){
  *       // Creates a range for every loop iteration
  *       // `loop_range` is nested inside `r`
- *       nvtx3::thread_range loop_range{};
+ *       nvtx3::scoped_range loop_range{};
  *    }
  * }
  * \endcode
  *
- * \subsection PROCESS_RANGE Process Range
+ * \subsection unique_range Process Range
  *
- * `nvtx3::domain_process_range` is identical to `nvtx3::domain_thread_range`
- * with the exception that a `domain_process_range` can be created and destroyed
+ * `nvtx3::unique_range_in` is identical to `nvtx3::scoped_range_in`
+ * with the exception that a `unique_range_in` can be created and destroyed
  * on different threads. This is useful to annotate spans of time that can
```

Comment:
See my comments for unique_range -- I think we should mention the other differences here too.